core/core/list_test.go

```diff
@@ -201,6 +201,27 @@ func TestGenerateControlRowsWithAllFields(t *testing.T) {
 	assert.Equal(t, want, got)
 }
 
+// Handles policies with no '|' characters in the string
+func TestGenerateControlRowsHandlesPoliciesWithEmptyStringOrNoPipesOrOnePipeMissing(t *testing.T) {
```

Comment:
Can you do a table-like test, with various policies lengths to test all cases?

Reply:
@matthyx I've enhanced the policy lists and added a new test-case with table-like structure to test pretty-print view of rows